lib/new_relic/agent/instrumentation/aws_sdk_lambda.rb

```diff
@@ -0,0 +1,28 @@
+# This file is distributed under New Relic's license terms.
+# See https://github.com/newrelic/newrelic-ruby-agent/blob/main/LICENSE for complete details.
+# frozen_string_literal: true
+
+DependencyDetection.defer do
+  named :aws_sdk_lambda
+
+  depends_on do
+    defined?(Aws::Lambda::Client)
+  end
+
+  executes do
+    require_relative 'aws_sdk_lambda/instrumentation'
+
+    # prepend_instrument and chain_instrument call extract_supportability_name
+    # to get the library name for supportability metrics and info-level logging.
+    # This is done by spliting on the 2nd to last spot of the instrumented
+    # module. If this isn't how we want the name to appear, pass in the desired
+    # name as a third argument.
```

Comment:
Do we need this comment? It doesn't look like a third argument is being passed.

Reply:
```suggestion
```
Got it. Mind if we remove it? Or do you prefer we keep it JIC a third arg is needed in the future?

Reply:
Boilerplate comment removed with [375db67](https://github.com/newrelic/newrelic-ruby-agent/pull/2926/commits/375db6736890c175e18f402a3239345310f03fec)

Reply:
Argh! Sorry, @kaylareopelle. I quickly skimmed the email notification saying "Mind if we remove it?" without noticing that you'd offered a suggested commit. Now in the web UI I see it clearly.

Reply:
That's okay, I don't feel offended! 😄  Good to know about how things render on email!